pkgs/games/steam/fhsenv.nix

```diff
@@ -249,6 +251,9 @@ in buildFHSUserEnv rec {
 
     ${exportLDPath}
     ${fixBootstrap}
+
+    set -o allexport # Export the following env vars
```

Comment:
Should we turn good off afterwards again? I am not sure if it is necessary.

Reply:
The shell script ends there with an exec, so I don't think that's necessary.